src/learn_to_pick/__init__.py

```diff
@@ -11,8 +11,6 @@
     ToSelectFrom,
     VwPolicy,
     VwLogger,
-    embed,
```

Comment:
You might want to use `base.embed` from an external featurizer (I used in in PyTorch featurizer for example)

Reply:
added back